src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -328,6 +328,108 @@ It is recommended to use this when reporting any issue.
 
 `subctl gather --module broker,service-discovery --type resources`
 
+### `cloud`
+
+#### `cloud prepare`
+
+`subctl cloud prepare [flags]`
+
+This command prepares the cloud for Submariner installation.
+
+#### `prepare aws` flags
```

Comment:
```suggestion
#### `prepare aws` 

`subctl cloud prepare aws [flags]`
```

Reply:
This is a convention that is followed on this page for other commands. Let me change it to [flags] as its more readable.